l1-contracts/contracts/common/libraries/openzeppelin/Arrays.sol

```diff
@@ -0,0 +1,487 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v5.0.0) (utils/Arrays.sol)
+// This file was procedurally generated from scripts/generate/templates/Arrays.js.
+
+pragma solidity ^0.8.20;
+
+import {SlotDerivation} from "./SlotDerivation.sol";
+import {StorageSlot} from "@openzeppelin/contracts/utils/StorageSlot.sol";
```

Comment:
most certainly it is safe to use v4 OZ library here in addition to copied v5 OZ library, but this should be warned about in the comments near the import

Reply:
This importing issue should not be pushed to main, we only need this while we did not import OZ-v5 and v4 in parallel. Once we merge that it will be removed